src/main/java/eu/foxxbl/x4/gameparser/shady/service/MapSectorService.java

```diff
@@ -41,9 +42,11 @@ private FactionEntity getFactionEntity(ParsedMapSector parsedMapSector, Map<Stri
   }
 
   private MapSectorEntity getMapSectorEntity(ParsedMapSector parsedMapSector, Map<String, MapSectorEntity> mapSectorMapByMacro) {
-    MapSectorEntity mapSectorEntity = mapSectorMapByMacro.get(parsedMapSector.sectorMacro());
+    String sectorMacro = parsedMapSector.sectorMacro();
+    MapSectorEntity mapSectorEntity = mapSectorMapByMacro.get(sectorMacro);
     if (mapSectorEntity == null) {
-      throw new RuntimeException("Cannot find translation for sector macro " + parsedMapSector.sectorMacro());
+        mapSectorEntity = new MapSectorEntity(sectorMacro, MapType.UNKNOWN, sectorMacro);
```

Comment:
could you please add a warn logging that it cannot find the translation for the sector macro, thanks!

Reply:
Sure; easily done.